webapp/src/components/sidebar_right/github_items.tsx

```diff
@@ -372,7 +372,7 @@ function getGithubLabels(labels: Label[]) {
     return labels.map((label) => {
         return (
             <Badge
-                key={label.id}
+                key={label.name}
```

Comment:
Why is there no `id` any longer?

Reply:
@hanzei We replaced the API calls to Github with GraphQL queries for fetching the data. The `id` returned from the GraphQL is an encrypted string. If we have to use that ID we will have to create a custom type on the server side for sending the data to frontend OR we need to explore how we can decrypt that id (Currently we are using the type from the GitHub package which has the id field of type integer). Since id is unnecessary in our use case, we used the `name ` field instead.

Reply:
Are label names unique in GitHub? i.e. do they make for a good `key`?

Reply:
@hanzei Yes, for a particular repo label name will be unique.
![image](https://github.com/mattermost/mattermost-plugin-github/assets/55234496/9b572eed-1ab8-4917-a210-5dedd61b30b7)